tests/codeceptjs/editors/advanced_test.js

```diff
@@ -8,4 +8,5 @@ Scenario('test validation & delete', (I) => {
     I.fillField('root[0][location][city]', 'Stuttgart');
     I.waitForValue("[name='root[0][location][city]']", 'Stuttgart');
     I.click('.json-editor-btntype-deletelast');
+    I.acceptPopup();
```

Comment:
To make it compatible with Puppeteer, you need to add this line before `I.click('.json-editor-btntype-deletelast');`


    I.amAcceptingPopups();

Reply:
Added, btw: do we need both?

Reply:
Yes unfortunately. The accept/cancel popups works differently in Puppeteer, where you have to place the amAcceptingPopups/amCancelingPopups BEFORE the click (Not very logic, but apparently the only way they could get it to work)